WordPress/src/main/java/org/wordpress/android/ui/reader/views/ReaderThumbnailStrip.java

```diff
@@ -61,19 +61,22 @@ private void initView(Context context) {
 
         int displayWidth = DisplayUtils.getDisplayPixelWidth(context);
         int margins = context.getResources().getDimensionPixelSize(R.dimen.reader_card_content_padding) * 2;
-        mThumbnailWidth = (displayWidth - margins) / IMAGE_COUNT;
+        mThumbnailWidth = (displayWidth - margins) / THUMBNAIL_STRIP_IMG_COUNT;
     }
 
     public void loadThumbnails(long blogId, long postId, boolean isPrivate) {
-        // get rid of any views already added
-        mView.removeAllViews();
```

Comment:
The only behavioural change in this class is that I moved `mView.removeAllViews();` below the `getImageList`. However, since the `getImageList` doesn't even work with views I believe it shouldn't change the behavior at all.